package.json

```diff
@@ -60,6 +60,7 @@
   },
   "dependencies": {
     "@hot-loader/react-dom": "16.11.0",
+    "@reduxjs/toolkit": "1.0.4",
```

Comment:
`@reduxjs/toolkit` includes redux-thunk by default. Not sure if webpack shakes-out this dependency if unused. IMO, toolkit is too opinionated (and hiding `immer` is too much "magic" for my taste). The ducks pattern can be implemented just as easily, and with a similar reduction in boilerplate, using a single-purpose library like `redux-actions`. Using `redux-actions` also allows easier transition to TypeScript, if desired (see [`typesafe-actions`](https://github.com/piotrwitek/typesafe-actions))